.github/workflows/ci.yml

```diff
@@ -60,28 +63,31 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
 
-      - name: Get Yarn cache path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Update npm
+        run: npm i -g npm@latest
```

Comment:
FWIW, I wouldn't do this...

Reply:
Ah yes we've discussed this before, I'll make it `npm@7` 👍